contracts/deployables/strategies/LinearERC20VotingV1.sol

```diff
@@ -44,7 +46,8 @@ contract LinearERC20VotingV1 is
      * Defines the current state of votes on a particular Proposal.
      */
     struct ProposalVotes {
-        uint48 votingStartTimestamp; // time that voting starts at
+        uint48 votingStartTimestamp; // time that voting starts
+        uint256 votingStartBlock; // block that voting starts
```

Comment:
as we talked about today we could replace these with just votingStartTimepoint

Reply:
not doing this